test/integration/multiplexed_upstream_integration_test.cc

```diff
@@ -534,13 +534,13 @@ TEST_P(MultiplexedUpstreamIntegrationTest, ManyResponseHeadersAccepted) {
   EXPECT_TRUE(response->complete());
 }
 
-// Tests that HTTP/2 response headers over 60 kB are rejected and result in a stream reset.
+// Tests that HTTP/2 response headers over 80KB are rejected and result in a stream reset.
```

Comment:
Please add a test that overrides the default value in the H2/H3 case.

Reply:
Added, Thanks!